server/controllers/emptyBuildings/post.js

```diff
@@ -0,0 +1,35 @@
+const { promisify } = require('util');
+const { reportBuilding } = require('../../models/queries/emptyBuildings');
+const { schemaBuildings } = require('../../validation');
+
+const postEmptyBuilding = async (req, res, next) => {
+  try {
+    const newBuild = await schemaBuildings.validate(req.body, {
+      abortEarly: false,
+    });
+    if (req.files) {
+      const file = req.files.filename;
```

Comment:
`req.files.filename` won't be called like this, but will be called as you send it.
I think we called it thumbnail and will be sent from the frontend like that
```suggestion
      const file = req.files.thumbnail;
```